squad/api/rest.py

```diff
@@ -1071,20 +1071,24 @@ class SuiteViewSet(viewsets.ModelViewSet):
     """
     Additional actions:
 
-     * `api/suites/<id>/testnames` GET
+     * `api/suites/<id>/tests` GET
 
-        Returns list of all test names belonging to this suite
+        Returns list of all test belonging to this suite
     """
 
     queryset = Suite.objects.all()
     serializer_class = SuiteSerializer
     filterset_class = SuiteFilter
     filter_class = filterset_class  # TODO: remove when django-filters 1.x is not supported anymore
 
-    @action(detail=True, methods=['get'], suffix='testnames')
-    def testnames(self, request, pk=None):
-        # In [4]: [t.name for t in SuiteMetadata.objects.filter(kind='test', suite='testsuite1')]
-        pass
+    @action(detail=True, methods=['get'], suffix='tests')
```

Comment:
IIRC the idea behind 'testnames' endpoint was to just present the list of names. This would be much faster than retrieving list of Test objects. I'm OK with adding both.